uwrt_mars_rover_drivetrain/uwrt_mars_rover_drivetrain_hw/drivers/roboteq_driver/src/CanopenInterface.cpp

```diff
@@ -96,7 +95,7 @@ CanopenInterface::CanopenInterface(canid_t roboteq_can_id, const std::string& if
   addr.can_family = AF_CAN;
   addr.can_ifindex = ifr.ifr_ifindex;
 
-  std::cout << ifname << " at index " << ifr.ifr_ifindex << std::endl;
+  ROS_DEBUG_STREAM(ifname << " at index " << ifr.ifr_ifindex);
```

Comment:
do we want these to be ROS_DEBUG_STREAM_NAMED?

Reply:
yeah it would be better if this was named. the problem is that this is outside of a node (its a library) and the library doesn't have reference to a name. i could refactor the library to take in a logger_name argument or something.

regardless though, i think its probably better off to address this in a separate pr. I'd say it'd make sense to tackle with #106. I think that our usage of rosconsole is a little careless rn and we should reserve it only for logging things that are tightly coupled with our ros nodes and anything related to ros. For things in hardware driver libraries that are directly associated with hw operation (like this one) or libraries that get used across our different ros nodes (like our can library), it really makes more sense to be publishing diagnostic messages imo(at least from lightly reading through the diagnostic stack documentation).